{
  "info": {
    "author": "Mohd. Shafikur Rahman",
    "author_email": "shafikshaon@gmail.com",
    "bugtrack_url": null,
    "classifiers": [
      "Environment :: Web Environment",
      "Intended Audience :: Developers",
      "License :: OSI Approved :: MIT License",
      "Operating System :: OS Independent",
      "Programming Language :: Python :: 2.7",
      "Programming Language :: Python :: 3"
    ],
    "description": "# django-debug-helper\nDjango debug error related exception search on Google/ Stack Overflow.\n\n### Installation\n```\n$ pip install django-debug-helper\n```\n\n### Setting it up\nAdd the custom middleware to your **MIDDLEWARE** in the ```settings.py```.\n```\nif DEBUG:\n\tMIDDLEWARE += ('django-debug-helper.debug.ErrorSearchMiddleware', )\n```\nNext time you get an error in your applications, you will see an external link to a custom **google/ stackoverflow** search as you defined.\n\n\nBy default it set to **google** search, but if you want to use **stackoverflow** search instead of **google** then add this into your ```settings.py```\n```\nDJANGO_TRACE_SEARCH_SITE = \"stackoverflow\"\n```\n\n\n\n",
    "description_content_type": "text/markdown",
    "docs_url": null,
    "download_url": "",
    "downloads": {
      "last_day": -1,
      "last_month": -1,
      "last_week": -1
    },
    "home_page": "https://github.com/shafikshaon/django-debug-helper",
    "keywords": "",
    "license": "MIT",
    "maintainer": "",
    "maintainer_email": "",
    "name": "django-debug-helper",
    "package_url": "https://pypi.org/project/django-debug-helper/",
    "platform": "",
    "project_url": "https://pypi.org/project/django-debug-helper/",
    "project_urls": {
      "Homepage": "https://github.com/shafikshaon/django-debug-helper"
    },
    "release_url": "https://pypi.org/project/django-debug-helper/1.1.5/",
    "requires_dist": null,
    "requires_python": "",
    "summary": "Django debug error related exception search on Google/ Stack Overflow.",
    "version": "1.1.5",
    "yanked": false,
    "yanked_reason": null
  },
  "last_serial": 4471849,
  "urls": [
    {
      "comment_text": "",
      "digests": {
        "blake2b_256": "23e3415c5c9d606a3a3ed0e8a0da2b0d859374d5e1ed91179a8c6302f29998d9",
        "md5": "096ad19a4a5541c778c04caae18df28c",
        "sha256": "1ded934c8e7c8d06ad9a87ae371e7bc62dfd079192ec35578109f2307167d309"
      },
      "downloads": -1,
      "filename": "django_debug_helper-1.1.5-py3-none-any.whl",
      "has_sig": false,
      "md5_digest": "096ad19a4a5541c778c04caae18df28c",
      "packagetype": "bdist_wheel",
      "python_version": "py3",
      "requires_python": null,
      "size": 4984,
      "upload_time": "2018-11-10T06:04:57",
      "upload_time_iso_8601": "2018-11-10T06:04:57.321496Z",
      "url": "https://files.pythonhosted.org/packages/23/e3/415c5c9d606a3a3ed0e8a0da2b0d859374d5e1ed91179a8c6302f29998d9/django_debug_helper-1.1.5-py3-none-any.whl",
      "yanked": false,
      "yanked_reason": null
    },
    {
      "comment_text": "",
      "digests": {
        "blake2b_256": "01474b69146939abfe678d5717f206acd3bedf4d733a79c7bf9358952f24b7f7",
        "md5": "8c5e1d7b1f213df2f9da86788f719c1c",
        "sha256": "78093fb652ee981c590f18e9866caaff46af5c8564c34b9c832dcb3d7a16a494"
      },
      "downloads": -1,
      "filename": "django-debug-helper-1.1.5.tar.gz",
      "has_sig": false,
      "md5_digest": "8c5e1d7b1f213df2f9da86788f719c1c",
      "packagetype": "sdist",
      "python_version": "source",
      "requires_python": null,
      "size": 2583,
      "upload_time": "2018-11-10T06:04:58",
      "upload_time_iso_8601": "2018-11-10T06:04:58.736572Z",
      "url": "https://files.pythonhosted.org/packages/01/47/4b69146939abfe678d5717f206acd3bedf4d733a79c7bf9358952f24b7f7/django-debug-helper-1.1.5.tar.gz",
      "yanked": false,
      "yanked_reason": null
    }
  ],
  "vulnerabilities": []
}